helpers/lbryApi.js

```diff
@@ -120,4 +122,22 @@ module.exports = {
         });
     });
   },
+  createChannel (name, claimAddress) {
+    return new Promise((resolve, reject) => {
+      axios
+        .post('http://localhost:5279/lbryapi', {
+          method: 'channel_new',
+          params: {
+            channel_name: name,
+            amount      : 0.1,
```

Comment:
Should also probably be a const, assuming this is the same value for channels and publishes.